qmxgraph/common_testing.py

```diff
@@ -32,3 +33,31 @@ def get_cell_ids(widget, filter_function):
         return cells.map(function(aCell){{ return aCell.getId(); }});
     }})()'''.format(filter_func=filter_function))
     return cells_ids
+
+
+@contextmanager
+def bridge_blocker(*bridge_signal_list):
+    """
+    Attach one callback blocker to each signal.
```

Comment:
Will it block until all signals are triggered, or unblock as the first one is triggered? Please add this to the docs.